server/routes/overview/overviewController.ts

```diff
@@ -128,7 +128,7 @@ export default class OverviewController {
     const { prisonNumber } = req.params
     const { prisonerSummary } = req.session
 
-    const timeline = await this.timelineService.getTimeline(prisonNumber, req.user.token)
+    const timeline = await this.timelineService.getTimeline(prisonNumber, req.user.token, req.user.username)
```

Comment:
We now also need to pass the username into the `TimelineService` method (see the change to `TimelineService` to see why)